update-dependencies/Invoke-UpdateDependencies.ps1

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+[cmdletbinding()]
+param(
+    [string]$UpdateDependenciesParams,
+    [switch]$CleanupDocker
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+$imageName = "update-dependencies"
+
+try {
+    & docker build -t $imageName -f $PSScriptRoot\Dockerfile --pull $PSScriptRoot
+    if ($LastExitCode -ne 0) {
```

Comment:
Not sure of the convention, but `LastExitCode`  should be all uppercase.

Reply:
This is the convention we have been using across the Docker repos so I would prefer to leave it for consistency.  I don't see this as a widely adopted convention.